src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double lowSide;
+    private final double topSide;
+    private final double lateralSide;
+
+    public IsoscelesTrapezoid(Color color, double lowSide, double topSide, double lateralSide) {
+        super(color);
+        this.lowSide = lowSide;
+        this.topSide = topSide;
+        this.lateralSide = lateralSide;
+    }
+
+    @Override
+    public void print() {
+        System.out.println("Figure: " + FigureName.ISOSCELES_TRAPEZOID.name()
+                + ", area:" + getArea()
+                + ", sq.units, low side:" + lowSide
+                + ", units, top side: " + lowSide
+                + ", units, lateral side: " + lateralSide
+                + ", units, color: " + getColor());
```

Comment:
[To get a string representation of an enum use `name()`](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants)